Python/Setup/swix/swix.generate.targets

```diff
@@ -49,10 +49,6 @@ package name=$([System.IO.Path]::GetFileNameWithoutExtension($(OutputName))) ver
 <WriteLinesToFile File="$(SwrFile).tmp" Lines="vs.dependencies
 $(_Dependencies)
 " Overwrite="no" Encoding="ASCII" Condition="$(_Dependencies) != ''" />
-
-    <ItemGroup>
-      <Package Include="$(SwrFile)" />
-    </ItemGroup>
   </Target>
 
   <Target Name="_GenerateSwrFromSources" Condition="@(SwrSource) != ''">
```

Comment:
I get MSBuild (we actually have a separate swixproj to handle files like that), but shouldn't Project / Item templates be deployed properly? Otherwise how does the experimental hive work?